### DISCUSSION TITLE
Ruby Package Manager overwrote packages already managed by Homebrew

### DISCUSSION THREAD

**Discussion by apprentice321** [Homebrew/discussions#34] [Everyday usage] [answered]
Downloaded Ruby Version Manager and ran rvm install ruby-2.7.2 to update ruby on MacOS. It ended up updating the entire packages inside the dir/usr/local/Cellar/ (already managed by the default package manager - homebrew). Anyway to recover without deleting the entire thing?

**Comment by SMillerDev** [comment, marked as answer]
Not sure what the ruby package manager is, but I think you should be reporting the issue with that project.

**Reply by apprentice321** [reply]
OK. You can delete this.
